specs/Button.md

```diff
@@ -43,7 +43,7 @@ The following are variants that exist because of the need of `Buttons` to reside
   - In Ant Design
   - In Semantic UI
   - In Shards React
-  - In Stardust UI
+  - In Current Fluent UI
```

Comment:
"Current Fluent" ... this makes no sense.

Reply:
@levithomason asked if I could do replace all the references to Stardust in the specs that's why I did this. I wasn't sure if simply saying Fluent UI would make sense since it's not the latest Fluent we're going towards so I tried this wording but I'm happy to change it to just Fluent.

Reply:
@dzearing is there an alternate proposal for @aneeshack4?